packages/react/src/PageHeader/PageHeader.docs.json

```diff
@@ -1,10 +1,10 @@
 {
   "id": "drafts_page_header",
   "name": "PageHeader",
-  "status": "draft",
+  "status": "alpha",
```

Comment:
Although we have usages in production (though I think they are only staff shipped), I think it is better to promote to Alpha first until we develop our confidence in the API. Let me know if you have any concerns 🙌🏻